### ISSUE TITLE
os.getlogin()

### ISSUE THREAD

**Issue by speaqr-ai** [Openmacro/openmacro#7]
raceback (most recent call last):
  File "/home/vince/.local/bin/macro", line 5, in <module>
    from openmacro.__main__ import main
  File "/home/vince/.local/lib/python3.10/site-packages/openmacro/__main__.py", line 2, in <module>
    from .core import Openmacro
  File "/home/vince/.local/lib/python3.10/site-packages/openmacro/core/__init__.py", line 1, in <module>
    from ..computer import Computer
  File "/home/vince/.local/lib/python3.10/site-packages/openmacro/computer/__init__.py", line 6, in <module>
    from ..utils import ROOT_DIR
  File "/home/vince/.local/lib/python3.10/site-packages/openmacro/utils/__init__.py", line 16, in <module>
    USERNAME = os.getlogin()
OSError: [Errno 6] No such device or address

i get this error while trying to use macro after it says, it installed. 
I am using elementary os, dont know if that may cause the problem.

**Comment by amooo-ooo** [comment]
Yeah, seems like a technical issue surrounding `glibc` with `os.getlogin()` on Linux based systems. I'll patch the problem soon, thanks for raising this 👍

**Comment by amooo-ooo** [comment]
Should be fixed, let me know if anything else is up 🙌
